conf.d/main

```diff
@@ -18,13 +18,38 @@ CONF=$WEBROOT/.env
 LOC_SHARE=/usr/local/share
 LOC_BIN=/usr/local/bin
 
+#install inoncube loader
+systemctl restart apache2
+wget http://downloads3.ioncube.com/loader_downloads/ioncube_loaders_lin_x86-64.tar.gz
+tar -xvf ioncube_loaders_lin_x86-64.tar.gz
+cp ioncube/ioncube_loader_lin_8.2.so /usr/lib/php/20220829
+
+#Update php related config files for ioncube loader
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/apache2/php.ini
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/cli/php.ini
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/fpm/php.ini
```

Comment:
Whilst it's not wrong (and some might even argue that it's "best practice") but we're using Apache's mod_php by default, so no need to tweak fpm config. FYI php-fpm is another/alternate way to serve/process PHP files which is webserver independent - i.e. works with Apache or other webservers - mod_php only works with Apache.